components/bt/host/bluedroid/Kconfig.in

```diff
@@ -144,6 +144,13 @@ config BT_HID_DEVICE_ENABLED
     help
         This enables the BT HID Device
 
+config BT_HID_DEVICE_TASK_SIZE
```

Comment:
I think this configuration option can be put into the esp_hid component and it would be better to have different configuration options for BLE HIDD and BR/EDR HIDD.

Reply:
I can separate these and put it in the esp_hid_component as its own kconfig